src/sentry/ingest/billing_metrics_consumer.py

```diff
@@ -167,7 +167,7 @@ def _produce_billing_outcomes(self, payload: MetricsBucket) -> None:
             key_id=None,
             outcome=Outcome.ACCEPTED,
             reason=None,
-            timestamp=datetime.fromtimestamp(payload["timestamp"]),
+            timestamp=datetime.now(),
```

Comment:
do you want to use `datetime.utcnow().replace(tzinfo=utc)` similar to https://github.com/getsentry/sentry/blob/81337eae77dbe3a4f3af88a35e292683b0b81d0b/src/sentry/relay/config/__init__.py#L221 ?
Since datetime.now() is just local time.

Reply:
from python docs:
> This function is preferred over [today()](https://docs.python.org/3/library/datetime.html#datetime.datetime.today) and [utcnow()](https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow).

Seems you are right, so ignore my comment.

Reply:
Good catch, thank you!